.azure-pipelines/common/build.yml

```diff
@@ -1,16 +1,16 @@
 steps:
-- task: NodeTool@0
-  displayName: 'Use Node 14.x'
-  inputs:
-    versionSpec: 14.x
+  - task: NodeTool@0
+    displayName: "Use Node 16.x"
+    inputs:
+      versionSpec: 16.x
 
-- task: Npm@1
-  displayName: 'npm ci'
-  inputs:
-    command: ci
+  - task: Npm@1
+    displayName: "npm ci"
+    inputs:
+      command: ci
 
-- task: Npm@1
-  displayName: 'Build'
-  inputs:
-    command: custom
-    customCommand: run build
```

Comment:
It's really small, but can you revert the inadvertent changes here. Mostly to preserve the git blame for this file.